python/scripts/worker-populate.py

```diff
@@ -41,13 +41,16 @@ def main(argv):
     run_daemon = True # execute at least one loop
     while run_daemon:
         populated_at_least_one = False
+        start = time.time()
         for name, rel in rels_cls.items():
             todo, done = rel().progress()
             if todo > 0:
                 populated_at_least_one = True
                 rel().populate(reserve_jobs=True)
```

Comment:
need to have the option to suppress errors.  populate does not catch errors by default.